keep/providers/sentry_provider/sentry_provider.py

```diff
@@ -88,6 +95,7 @@ def __init__(
         super().__init__(context_manager, provider_id, config)
         self.sentry_org_slug = self.config.authentication.get("organization_slug")
         self.project_slug = self.config.authentication.get("project_slug")
+        self.SENTRY_API = self.config.authentication.get("api_url", self.SENTRY_API)
```

Comment:
This is problematic as it will change API_URL constantly (for example, someone might change this to self hosted and override in general provider)
```suggestion
        self.SENTRY_API = self.config.authentication.get("api_url", self.SENTRY_API)
```

Reply:
So do you mean it can affect other connected sentry providers?

Reply:
That’s right. `__init__` should declare self.sentry_api and adjust functions to use it.

Reply:
Would probably rename the const to `SENTRY_MANAGED_API`